app/app.go

```diff
@@ -147,8 +147,7 @@ import (
 
 const (
 	appName                        = "kava"
-	FixDefaultAccountUpgradeHeight = 138592
-	EthermintPatchUpgradeName      = "v0.18.0"
+	FixDefaultAccountUpgradeHeight = 1783387
```

Comment:
updated to match this commit @galxy25 https://github.com/Kava-Labs/kava/pull/1313/commits/34dccb9775b9709bc544016e70fbdfe91c0d28b1

Reply:
@DracoLi was wondering if that commit was for a mainnet block height (cc @rhuairahrighairidh ), since the hotfix that commit was associated with has already been deployed to testnet should we keep the same `FixDefaultAccountUpgradeHeight`?